service_mesh/v1x/ossm-security.adoc

```diff
@@ -11,4 +11,4 @@ include::modules/ossm-security-mtls.adoc[leveloffset=+1]
 
 include::modules/ossm-security-cipher.adoc[leveloffset=+1]
 
-include::modules/ossm-security-cert-manage.adoc[leveloffset=+1]
+include::modules/ossm-security-cert-manage-1x.adoc[leveloffset=+1]
```

Comment:
Why is this ossm-security-cert-manage-1x.adoc and not ossm-security-cert-manage.adoc?

Reply:
I figure that checking in changes like this can help reduce manual cherry picks in the future.

Reply:
@JStickler I think you're right. This works for me.